Traducir.Web.Net/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,105 @@
+            </button>
+            <div class="collapse navbar-collapse">
+                <ul class="ml-auto navbar-nav">
+                    <li class="nav-item">
+                        <a href="https://github.com/g3rv4/Traducir" target="_blank" class="nav-link">Source Code</a>
+                    </li>
+                    <li class="nav-item dropdown">
+                        <a href="#" class="dropdown-toggle nav-link" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                            Database
+                        </a>
+                        <div tabindex="-1" role="menu" aria-hidden="true" class="dropdown-menu dropdown-menu-right">
+                            <a class="dropdown-item" href="https://db.traducir.win">Access the Database</a>
+                            <a class="dropdown-item" href="https://github.com/g3rv4/Traducir/blob/master/docs/USING_REDASH.md">Instructions</a>
+                        </div>
+                    </li>
+                    @if (viewModel.IsLoggedIn)
+                    {
+                        <li class="nav-item dropdown">
+                            <a href="#" class="dropdown-toggle nav-link" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                My account
+                            </a>
+                            <div tabindex="-1" role="menu" aria-hidden="true" class="dropdown-menu dropdown-menu-right">
+                                <a class="dropdown-item" href="suggestions/@(viewModel.UserId)">My Suggestions</a>
+                                <a class="dropdown-item" href="notifications">My Notifications</a>
+                            </div>
+                        </li>
+                        <li class="nav-item">
+                            <a href="users" target="_blank" class="nav-link">Users</a>
+                        </li>
+                    }
+                    <li class="nav-item">
+                        @{
+                            var returnUrl = System.Net.WebUtility.UrlEncode(viewModel.CurrentPathAndQuery);
+                        }
+                        @if (viewModel.IsLoggedIn)
+                        {
+                            <a href="/app/logout?returnUrl=@returnUrl" class="nav-link">Log out</a>
+                        }
+                        else
+                        {
+                            <a href="/app/login?returnUrl=@returnUrl" class="nav-link">Log in!</a>
+                        }
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    </div>
+
+    @RenderBody()
+
+    <environment include="Development">
+        <script src="~/lib/jquery/dist/jquery.js"></script>
+        <script src="~/lib/bootstrap/dist/js/bootstrap.bundle.js"></script>
+    </environment>
+    <environment exclude="Development">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"
```

Comment:
I dream of a world where we don't use jquery at all...

Reply:
This was added automatically by Visual Studio when creating the project. For now I've left it as is because I was not sure if it was needed by Bootstrap, but don't worry, I don't plan to use it in my own JS code.